ejb-txn-remote-call/README.adoc

```diff
@@ -0,0 +1,942 @@
+[source,options="nowrap"]
+----
+curl -s http://localhost:8080/client/remote-outbound-fail-stateless
+----
++
+. Check the server logs
+.. The `server1` will contain error
++
+[source,options="nowrap"]
+----
+ARJUNA016036: commit on < formatId=131077, gtrid_length=35, bqual_length=36, tx_uid=..., node_name=server1, branch_uid=..., subordinatenodename=null, eis_name=unknown eis name > (Subordinate XAResource at remote+http://localhost:8180) failed with exception $XAException.XA_RETRY: javax.transaction.xa.XAException: WFTXN0029: The peer threw an XA exception
+----
++
+. The `server2` or `server3` will contain error
++
+[source,options="nowrap"]
+----
+ARJUNA016036: commit on < formatId=131077, gtrid_length=35, bqual_length=43, tx_uid=..., node_name=server1, branch_uid=..., subordinatenodename=server2, eis_name=unknown eis name > (org.jboss.as.quickstarts.ejb.mock.MockXAResource@731ae22) failed with exception $XAException.XAER_RMFAIL: javax.transaction.xa.XAException
+----
++
+. Verify number of commits done for the test XAResource at the server side
++
+[source,options="nowrap"]
+----
+curl -s http://localhost:8180/server/commits
+curl -s http://localhost:8280/server/commits
+----
++
+. Start recovery processing by entering `SCAN` string with command `telnet localhost 4712` at terminal
++
+[NOTE]
+====
+If the telnet is not available at your machine is recommended to be installed for you may proceed
+further with this guide.
+====
++
+.. After the recovery is processed the server log at `server2` or `server3` should contain warning
++
+[source,options="nowrap"]
+----
+WARN  [com.arjuna.ats.jta] (Periodic Recovery) ARJUNA016114: Could not load org.jboss.as.quickstarts.ejb.mock.MockXAResource will try to get XAResource from the recovery helpers
+----
++
+. Verify the number of commits done for the test XAResource at the server side. The commit count should be increased by one.
++
+[source,options="nowrap"]
+----
+curl -s http://localhost:8180/server/commits
+curl -s http://localhost:8280/server/commits
+----
+
+include::../shared-doc/undeploy-the-quickstart.adoc[leveloffset=+1]
+
+Reapeat the same for the `server2` and `server3` by navigating
+to the quickstart subfolder ``${PATH_TO_QUICKSTART_DIR}/ejb-txn-remote-call/server` and running
+
+[source,sh,options="nowrap"]
+----
+mvn wildfly:undeploy -Dwildfly.port=10090
+mvn wildfly:undeploy -Dwildfly.port=10190
+----
+
+== Server Log: Expected Warnings and Errors
+
+You will see the following warnings in the server log. You can ignore these warnings.
+
+[source,options="nowrap"]
+----
+HHH000431: Unable to determine H2 database version, certain features may not work
+
+WFLYDM0111: Keystore standalone/configuration/application.keystore not found, it will be auto generated on first use with a self signed certificate for host localhost
+
+WFLYSRV0018: Deployment "deployment.server.war" is using a private module ("org.jboss.jts") which may be changed or removed in future versions without notice.
+----
```

Comment:
At what point in the quickstart can I expect to see these warnings. Maybe there needs to be a link to this section from the step that generates the warning or error.

Reply:
+1, I added the information that the warnings are shown during the server startup.